ui.config/src/main/content/jcr_root/apps/wknd/osgiconfig/config.author/com.adobe.granite.csrf.impl.CSRFFilter.cfg.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Similar to CORS, at a glance, this appears to be the same for `config.author` and `config.publish` .. any reason not to consolidate down to `config`?

(unless I'm missing  something in am eye-ball comparison)

Reply:
AEM provides a default CSRFFilter so this is the only way AFAIK to ensure it gets overwritten (since this is a more specific config)